src/Responses/Threads/Messages/Delta/ThreadMessageDeltaResponseContentText.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenAI\Responses\Threads\Messages\Delta;
+
+use OpenAI\Contracts\ResponseContract;
+use OpenAI\Responses\Concerns\ArrayAccessible;
+use OpenAI\Testing\Responses\Concerns\Fakeable;
+
+/**
+ * @implements ResponseContract<array{value: string, annotations: array<int, array{type: string, text: string, file_citation: array{file_id: string, quote: string}, start_index: int, end_index: int}|array{type: string, text: string, file_path: array{file_id: string}, start_index: int, end_index: int}>}>
+ */
+final class ThreadMessageDeltaResponseContentText implements ResponseContract
+{
+    /**
+     * @use ArrayAccessible<array{value: string, annotations: array<int, array{type: string, text: string, file_citation: array{file_id: string, quote: string}, start_index: int, end_index: int}|array{type: string, text: string, file_path: array{file_id: string}, start_index: int, end_index: int}>}>
+     */
+    use ArrayAccessible;
+
+    use Fakeable;
+
+    /**
+     * @param  array<int, ThreadMessageResponseContentTextAnnotationFilePathObject|ThreadMessageResponseContentTextAnnotationFileCitationObject>  $annotations
+     */
+    private function __construct(
+        public string $value,
+        public ?array $annotations,
+    ) {
+    }
+
+    /**
+     * Acts as static factory, and returns a new Response instance.
+     *
+     * @param  array{value: string, annotations: array<int, array{type: 'file_citation', text: string, file_citation: array{file_id: string, quote: string}, start_index: int, end_index: int}|array{type: 'file_path', text: string, file_path: array{file_id: string}, start_index: int, end_index: int}>}  $attributes
+     */
+    public static function from(array $attributes): self
+    {
+        if (isset($attributes['annotations'])) {
+            $annotations = array_map(
+                fn (array $annotation): ThreadMessageResponseContentTextAnnotationFileCitationObject|ThreadMessageResponseContentTextAnnotationFilePathObject => match ($annotation['type']) {
+                    'file_citation' => ThreadMessageResponseContentTextAnnotationFileCitationObject::from($annotation),
+                    'file_path' => ThreadMessageResponseContentTextAnnotationFilePathObject::from($annotation),
+                },
+                $attributes['annotations'],
+            );
+        }
+
+        return new self(
+            $attributes['value'],
```

Comment:
`value` here could be null, the current implementation throws an error `Undefined array key 'value'` 

```json
{
   "id":"msg_SB3FBqivUlwSkKgC7oUCk2LX",
   "object":"thread.message.delta",
   "delta":{
      "content":[
         {
            "index":0,
            "type":"text",
            "text":{
               "annotations":[
                  {
                     "index":0,
                     "type":"file_path",
                     "text":"sandbox:/mnt/data/example.xlsx",
                     "start_index":198,
                     "end_index":256,
                     "file_path":{
                        "file_id":"file-xxxxxxxx"
                     }
                  }
               ]
            }
         }
      ],
      "file_ids":[
         "file-xxxxxxxxxxx"
      ]
   }
}
```